sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/ITagsOperationOperations.cs

```diff
@@ -0,0 +1,299 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.ResourceManager
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Models;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// TagsOperationOperations operations.
+    /// </summary>
+    public partial interface ITagsOperationOperations
```

Comment:
>ublic partial interface ITagsOperationOperations [](start = 5, length = 48)

After the changes we reverted on Tags, I'd think the name of the interface would be "ITagsOperations" - could you check on this please?

Reply:
I think this is leftover from the time we generated the SDK before reverting those changes. I don't think it's being used anywhere (I see that ITagsOperations exists separately), I should be able to just remove it. I will look into it.

Reply:
Just looked into it and yes, they were leftover from previous SDK generation as the TagsOperations files did not replace them. The class are not referenced anywhere outside their own definitions/extensions and build/tests pass successfully even on removing them.